source/dagger/main.cpp

```diff
@@ -1,14 +1,16 @@
 
 #include "core/engine.h"
-#include "gameplay/ping_pong/ping_pong_main.h"
-#include "gameplay/platformer/platformer_main.h"
-#include "gameplay/racing/racing_main.h"
-#include "gameplay/team_game/team_game_main.h"
+//#include "gameplay/ping_pong/ping_pong_main.h"
+//#include "gameplay/platformer/platformer_main.h"
+//#include "gameplay/racing/racing_main.h"
+//#include "gameplay/team_game/team_game_main.h"
+#include "gameplay/tiles_example/tiles_example_main.h"
```

Comment:
why do u need to comment everything else here? 
it'll make enabling of other examples a bit harder

Reply:
I just wanted to get the game to load faster :D we can get this out, really